### ISSUE TITLE
Website improvements for GEC

### ISSUE THREAD

**Issue by cticenhour** [shannon-lab/zapdos#41]
This is a general issue/task list regarding the Zapdos website, now that we have something live and are iterating on it up to GEC. Feel free to edit this and add tasks. Please reference this issue as you make PRs.

- [ ] Instructions for Peacock in Getting Started instructions
- [ ] Benchmark pages (links added to main navigation?)
- [ ] Central location for talks given based on Zapdos?
- [ ] Contribution Guidelines
- [ ] Code Standards
- [ ] Zapdos doxygen?

**Comment by cticenhour** [comment]
For Doxygen, we might be able to use an action: https://github.com/marketplace/actions/doxygen-action

**Comment by cticenhour** [comment]
A possible improvement for deploying the website (rather than using the manual git commands we currently use): https://github.com/marketplace/actions/github-pages-action

**Comment by csdechant** [comment]
@cticenhour what did you mean by:
> Central location for talks given based on Zapdos?

Do you mean something like the current [List of Publications](https://shannon-lab.github.io/zapdos/publications.html) page but for previous conference talks?

**Comment by cticenhour** [comment]
This idea sprang from a desire to make our conference publications more accessible. At the time, I was interested in having them hosted somewhere (Zenodo or similar) and then our site could link out to them. At this point, that might be a Herculean effort. I would not mind just limiting ourselves to journal articles, theses, and conference publications that are actual papers (like ANS submissions, as an example).

**Comment by csdechant** [comment]
> At this point, that might be a Herculean effort.

I don't think it was be that hard, since the pool of people that have used Zapdos at conferences are still relativity small (for example, I personal have a digital archive of all my conference poster and presentations).

That said, I honestly don't think that linking to old presentations would bring a lot of value. In my view, conference presentations are like a pre-vetting process for information before it makes it into a paper. So, the information from old presentations would be distilled into a paper (or at the very least, we might at it as a V&V or showcase example on the website).

I still like this idea, but just for newer presentations, say within the past year or so, where the information is only in that presentation.

**Comment by cticenhour** [comment]
> In my view, conference presentations are like a pre-vetting process for information before it makes it into a paper.

I agree wholeheartedly with this statement. Before we had papers, beyond Alex's, I would have said the conference submissions were more important and needed to be highlights. Now I don't really care. Let's just stick with journal papers, theses, and (potentially) technical reports if we get to that point at INL.
